scripts/slack/notify_success_no_release.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+
+import requests
+import json
+
+circle_build_url = os.getenv('CIRCLE_BUILD_URL')
+url = os.getenv('SLACK_WEBHOOK')
+
+data = {
+  'attachments': 
+  [
+    {
+      'color': '#7CD197',
+      'fallback': 'Build Notification: ' + circle_build_url,
+      'title': 'Kubernetes-Monitor Publish Notification',
+      'text': ':egg_fancy: Successful `staging` merge, but no semantic-release occurring :egg_fancy:'
+    }
+  ]
+}
+
+requests.post(url, data = json.dumps(data))
```

Comment:
are you sure we need the ```json.dumps``` here? and in the other places
https://www.w3schools.com/python/ref_requests_post.asp

Reply:
Hmm maybe not! Looks like I can send it a json parameter, looking into it now